woodwork/tests/utils/test_accessor_utils.py

```diff
@@ -229,3 +233,19 @@ def test_is_schema_valid_false(sample_df):
 
     missing_col_df = sample_df.drop(columns={'is_registered'})
     assert not is_schema_valid(missing_col_df, schema)
+
+
+def test_is_dask_dataframe(sample_df_dask):
+    assert _is_dask_dataframe(sample_df_dask)
+
+
+def test_is_dask_series(sample_series_dask):
+    assert _is_dask_series(sample_series_dask)
+
+
+def test_is_koalas_dataframe(sample_df_koalas):
+    assert _is_koalas_dataframe(sample_df_koalas)
+
+
+def test_is_koalas_series(sample_series_koalas):
+    assert _is_koalas_series(sample_series_koalas)
```

Comment:
I think it would also be good to test that `False` is returned if we pass in pandas input to these functions:
```
assert not _is_dask_dataframe(pd.DataFrame())
```